### PR TITLE
Add draft for conference submission form

### PR DESCRIPTION
**Pull request by jorgensd**
Not ready to be merged yet, as I need to figure out the conference payment system.
Any feedback on the registration form and site layout is welcomed.

### CODE REVIEW

**Comment by francesco-ballarin** [comment]
You may want to allow anyone to see (and submit) the registration form. I tried opening the link from a chrome incognito window and it asks me to log in with a google account.

**Comment by jorgensd** [comment]
> You may want to allow anyone to see (and submit) the registration form. I tried opening the link from a chrome incognito window and it asks me to log in with a google account.

Changed setting now

**Comment by francesco-ballarin** [comment]
Confirmed. I went through the registration form (without submitting it). The only comment I have is on

```
Are you a PhD student or a Postdoctoral Researcher?
```

Make the third option to be "None of the above" instead of others? We don't really need to know if the submitter is a professor or any other role, we only care about this info for the award, right?

**Comment by jorgensd** [comment]
> Confirmed. I went through the registration form (without submitting it). The only comment I have is on
> 
> ```
> Are you a PhD student or a Postdoctoral Researcher?
> ```
> 
> Make the third option to be "None of the above" instead of others? We don't really need to know if the submitter is a professor or any other role, we only care about this info for the award, right?

I guess it would be nice for scoping the conference attendees, but it is probably not allowed due to GDPR.

File: conference/registration_form.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Talk and poster submission
```
Comment (mscroggs) [inline conference/registration_form.md:2]:
```suggestion
title: Talk and poster submission
permalink: /fenics-2024/submit/
```

**Comment by jorgensd** [comment]
@meg-simula here are some screenshots of a local build of the webpage:
image
image
image

**Comment by meg-simula** [comment]
Looks great @jorgensd! Perhaps we could add some information about the proceedings (to be)? A separate section on the FEniCS 2024 conference web page?

File: conference/2024.md [hunk 2]
```diff
@@ -24,7 +24,23 @@ and dedicated time for discussions and coding.
 
 Links to previous FEniCS conferences can be found [here](index.md).
 
-More information about the conference will appear here soon.
+## Abstract Submission
+
+Please submit your abstract by **April 5th** by filling in the [abstract submission form](pages_2024/registration_form.md).
+
```
Comment (meg-simula) [inline conference/2024.md:30]: Add about the conference proceedings here? Feel free to use info from our shared doc.

**Pull request merged**